src/gitHubIssueFetcher.ts

```diff
@@ -99,12 +100,14 @@ export class GitHubIssueFetcher {
         const pageSize = 100;
         const numberOfReposPerCall = this.config.numberOfReposPerCall;
         const reposForEachCall = this.splitArray(repositoryIdentifiers, numberOfReposPerCall);
+        const daysAgoCreated = this.config.daysAgoCreated != undefined ? this.config.daysAgoCreated : 90;
+        const dateStringToQuery = DateTime.local().minus({ days: daysAgoCreated }).toISODate();
```

Comment:
DateStringToQuery almost sounds like a conversion (XToY). Perhaps DateStringForQuery would be clearer?

Reply:
Good point. Changing that.
I also thought about simply DateString, but decided more descriptive was slightly better?
I'm just going to use DateStringForQuery so that I can merge it in right away, but am curious which you think is better.

Reply:
I like the more descriptive name. I had a separate comment about the capital D, and now I see that it might have come from what I typed in my earlier comment. Oops.